sympy/solvers/solveset.py

```diff
@@ -2760,16 +2771,23 @@ def linsolve(system, *symbols):
     >>> linsolve([], x)
     EmptySet
 
-    * An error is raised if, after expansion, any nonlinearity
-      is detected:
+    * An error is raised if any nonlinearity is detected, even
+    if it could be removed with expansion
```

Comment:
I'm guessing that the indentation here is the sphinx problem:
```
/home/runner/work/sympy/sympy/sympy/solvers/solveset.py:docstring of sympy.solvers.solveset.linsolve:169: WARNING: Bullet list ends without a blank line; unexpected unindent.
```

Reply:
I used the editor to search for leading space followed by `*`, looking for this case. Not sure how I missed it! Thanks.